web/vibe/web/rest.d

```diff
@@ -1603,12 +1603,14 @@ private HTTPServerRequestDelegate jsonMethodHandler(alias Func, size_t ridx, T)(
 				immutable serializer_ind = get_matching_content_type!(result_serializers)(accept_str);
 				foreach (i, serializer; result_serializers)
 					if (serializer_ind == i) {
-						auto serialized_output = appender!string;
-						static if (!__traits(compiles, () @safe {
-							serializer.serialize(serialized_output, ret);
-						}))
+						auto serialized_output = appender!(ubyte[]);
+						static if (
+							__traits(compiles, () @trusted { serializer.serialize(serialized_output, ret); })
+							&& !__traits(compiles, () @safe { serializer.serialize(serialized_output, ret); }))
+						{
 							pragma(msg, "Non-@safe serialization of REST return types deprecated - ensure that " ~
```

Comment:
I really hate this BTW :D

Reply:
Looks like a candidate for removal for 1.0.0

Reply:
I mean, the fact that `@safe` is forced on the user :/

Reply:
That's very much as intended, because we are talking about a REST service that has no business in deliberately choosing the memory-unsafe mode by default. If someone really wants it that way, it's always possible to slap on a `@trusted`.

Reply:
Well, I don't think we should force a different default on the user.
My understanding is that was put in place because we had to make a choice: be usable from `@safe` code, or not. You can't really choose to support both at the moment, which is a huge PITA to write libraries. So I understand why it's seen as preferable to support `@safe`, but it's quite annoying to force this on the user.

Re: slapping `@trusted`, what's the point of attributes if you just bypass it ?

Anyway, that's slightly OT.